package/network/utils/iproute2/Makefile

```diff
@@ -122,26 +125,29 @@ define Build/Configure
 endef
 
 TARGET_CFLAGS += -ffunction-sections -fdata-sections -flto
-TARGET_LDFLAGS += -Wl,--gc-sections
+TARGET_LDFLAGS += -Wl,--gc-sections -Wl,--as-needed
 TARGET_CPPFLAGS += -I$(STAGING_DIR)/usr/include/libnl-tiny
 
 MAKE_FLAGS += \
 	KERNEL_INCLUDE="$(LINUX_DIR)/user_headers/include" \
 	SHARED_LIBS=$(SHARED_LIBS) \
 	IP_CONFIG_TINY=$(IP_CONFIG_TINY) \
+	LIBBPF_FORCE=$(LIBBPF_FORCE) \
 	HAVE_ELF=$(HAVE_ELF) \
 	HAVE_MNL=$(HAVE_MNL) \
 	HAVE_CAP=$(HAVE_CAP) \
 	IPT_LIB_DIR=/usr/lib/iptables \
 	XT_LIB_DIR=/usr/lib/iptables \
-	FPIC="$(FPIC)"
+	FPIC="$(FPIC)" \
+	$(if $(findstring c,$(OPENWRT_VERBOSE)),V=1,V='')
 
 define Build/Compile
 	+$(MAKE_VARS) $(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) $(MAKE_FLAGS)
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/usr/include/iproute2
+	$(CP) $(PKG_BUILD_DIR)/include/bpf_elf.h $(1)/usr/include/iproute2
```

Comment:
Why did you change the files which are copied over?

Reply:
It's a missing development header (bpf_elf.h), previously overlooked, that helps with BPF map definitions typically used with tc. It's mentioned in the commit message too (though not as detailed) and the existing headers are still copied. What was the concern exactly?

Reply:
To avoid any doubt, it bears mentioning that the previous install directories and header files do not change; they are still installed to the same locations with:
```
$(CP) $(PKG_BUILD_DIR)/include/{libgenl,libnetlink}.h $(1)/usr/include/
```

The change to `$(INSTALL_DIR) $(1)/usr/include/iproute2` only creates an additional subdirectory for the `bpf_elf.h` header, supporting the `tc` BPF loader. This header install location is standard with upstream `iproute2` and also used in some other distro packages e.g. Ubuntu, Alpine, Arch. There's also some [related Debian discussion](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=904122#35). 

Hope that clarifies things.

Reply:
thanks for the clarification.